perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ModulesUtilsBlImpl.java

```diff
@@ -1043,14 +1043,14 @@ public User getUserFromMessage(PerunSessionImpl sess, String message) throws Int
 			try {
 				user = getPerunBl().getUsersManagerBl().getUserById(sess, userExtSource.getUserId());
 			} catch (UserNotExistsException ex) {
-				log.debug("User from UserExtSource {} can't be found by id in Perun. Probably not exists any more or UserExtSource was audited without proper UserId!", userExtSource);
+				log.info("User from UserExtSource {} can't be found by id in Perun. Probably not exists any more or UserExtSource was audited without proper UserId!", userExtSource);
```

Comment:
I don't see any reason, why this should be the info level.

Reply:
I think it should be warn level here, because it parses a user from an audit message, but the user no longer exists.

Reply:
Please skip this part, we will focus on this method separately. There is need for refactoring of this try catch block. Now there is debug info and also error info in one branch so it is confusing.